### PR TITLE
fix: outdated docs for loss functions

### PR DESCRIPTION
**Pull request by anaprietonem**
## Description
https://github.com/ecmwf/anemoi-core/issues/506

## What problem does this change solve?
Outdated docs about loss function.
##  Additional notes ##

***As a contributor to the Anemoi framework, please ensure that your changes include unit tests, updates to any affected dependencies and documentation, and have been tested in a parallel setting  (i.e., with multiple GPUs). As a reviewer, you are also responsible for verifying these aspects and requesting changes if they are not adequately addressed. For guidelines about those please refer to https://anemoi.readthedocs.io/en/latest/***

By opening this pull request, I affirm that all authors agree to the [Contributor License Agreement.](https://github.com/ecmwf/codex/blob/main/Legal/contributor_license_agreement.md)

----
📚 Documentation preview 📚: https://anemoi-training--525.org.readthedocs.build/en/525/

----
📚 Documentation preview 📚: https://anemoi-graphs--525.org.readthedocs.build/en/525/

----
📚 Documentation preview 📚: https://anemoi-models--525.org.readthedocs.build/en/525/

### CODE REVIEW

File: training/docs/modules/losses.rst [hunk 1]
```diff
@@ -25,13 +25,17 @@ in the config file at ``config.training.training_loss``, and
...
--  ``WeightedLogCoshLoss``: Latitude-weighted log-cosh loss.
--  ``WeightedRMSELoss``: Latitude-weighted root-mean-squared-error.
+-  ``MSELoss``: mean-squared-error.
+-  ``RMSELoss``: root mean-squared-error.
+-  ``MAELoss``: mean-absolute-error.
+-  ``HuberLoss``: Huber loss.
+-  ``LogCoshLoss``: log-cosh loss.
 -  ``CombinedLoss``: Combined component weighted loss.
 
+All the above losses by default are averaged across the grid nodes. To
```
Comment (anaprietonem) [inline training/docs/modules/losses.rst]: @JPXKQX do you agree with this?

Comment (JPXKQX) [reply training/docs/modules/losses.rst]: The loss is scaled (if specified) first. Then it is summed over the grid dimension and averaged along the batch and ensemble dimensions.
[code](https://github.com/ecmwf/anemoi-core/blob/main/training/src/anemoi/training/losses/base.py#L174)

**Review by JPXKQX — APPROVED** [review]
Looks good! Thanks
